src/sentry/plugins/interfaces/releasehook.py

```diff
@@ -88,5 +89,26 @@ def finish_release(self, version, **values):
             datetime=values['date_released'],
         )
 
+        # check if user exists, and then try to get refs based on version
+        if values.get('owner', None):
```

Comment:
alternatively, you could define a `get_repo_option_key` method that would return `'heroku:repository'`. on the heroku release hook, and `None` in the base class. basically, we just want to move anything heroku specific into sentry-plugins